couchbase.lite.console/couchbase.lite.console.csproj

```diff
@@ -1,9 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp1.1</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Can we bumb the version to 3.0? Thanks!

Reply:
This will make it incompatible with VS2017, are you ok with that?